e2e/fixtures/dotnet/scripting/build.cake

```diff
@@ -0,0 +1,20 @@
+// @OctoLinkerResolve(https://www.nuget.org/packages/Cake.DotNetTool.Module)
+#module nuget:?package=Cake.DotNetTool.Module&version=0.4.0
+
+// @OctoLinkerResolve(https://www.nuget.org/packages/Cake.Coveralls)
+#addin "nuget:https://api.nuget.org/v3/index.json?package=Cake.Coveralls&version=0.10.1"
```

Comment:
@stefanbuck I'm having trouble getting these lines to work as I wanted.

I was trying to make everything after `#module` & `#addin` the link so you didn't have to search for the package name in the url, but GitHub's parsing the lines with a full url and everything after `//` is in a span so no link gets injected.

Is there a way to support making that whole value a link or are we stuck with making just the package name the link?

This is the file I was testing against https://github.com/cake-build/cake/blob/develop/build.cake.

Reply:
This sounds similar to an issue I'm currently facing getting PHP / laravel support in. 

```html
<td>
  <span class="pl-k">use</span><span data-rgh-whitespace="space"> </span><span class="pl-v">Illuminate</span>\<span class="pl-v">Contracts</span>\<span class="pl-v">Container</span>\<span class="pl-v">Container</span>;
</td>
```

Yesterday, I spent two hours try to come up with a solution without luck. It's mainly a lack of support for such markup in https://github.com/padolsey/findAndReplaceDOMText which we use in the `insert-link` package. 

I've noticed that [refined-github](https://github.com/sindresorhus/refined-github) is able to wrap the url in your case. If we find a solution we should make sure it doesn't conflict with `refined-github` which is a widely used extension as well. 

Is this a blocker for you? Maybe best to ignore it for now and investigate into it separately.

Reply:
No not a blocker. I can have just the package name be the link until we find a better solution.